### PR TITLE
Fix valory app naming

### PR DESCRIPTION
**Pull request by jmoreira-valory**
## Proposed changes

Fix Valory app naming in various files.

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,25 +40,20 @@
   </a>
 </p>
 
-Open Autonomy is a framework for the creation of Agent Services: off-chain services which run as a multi-agent-system (MAS)  and are replicated on a temporary consensus gadget (blockchain) while being crypto-economically secured on a public blockchain, hence offering robustness, transparency and decentralization off-chain.
+**Open Autonomy** is a framework for the creation of **agent services**: off-chain
```
Comment (angrybayblade) [inline README.md:43]: I think it should be `autonomous service` in this case

File: README.md [hunk 2]
```diff
@@ -40,25 +40,20 @@
...
+autonomous services which run as a multi-agent-system (MAS) and offer enhanced functionalities
+on-chain. Agent services expand the range of operations that traditional
+smart contracts provide, making it possible to execute **arbitrarily complex operations**
+(such as machine-learning algorithms). Most importantly, agent services are
+**decentralized**, **trust-minimized**, **transparent**, and **robust**.
 
-As opposed to traditional smart contracts, Valory apps go beyond simple, purely on-chain logic without giving up on decentralization.
 
-## Get started developing autonomous services
+## Get started developing agent services
```
Comment (angrybayblade) [inline README.md:51]: Same as above

File: autonomy/data/Dockerfiles/dev/start_dev.sh [hunk 3]
```diff
@@ -4,7 +4,7 @@ sudo chown -R ubuntu:ubuntu /benchmarks
 sudo chown -R ubuntu:ubuntu /home/ubuntu
 sudo chown -R ubuntu:ubuntu /home/ubuntu/packages
 
-echo "Starting Dev image and installing Valory app dependencies."
+echo "Starting Dev image and installing agent service dependencies."
```
Comment (angrybayblade) [inline autonomy/data/Dockerfiles/dev/start_dev.sh:7]:
```suggestion
echo "Starting Dev image and installing agent dependencies."
```

File: autonomy/deploy/base.py [hunk 4]
```diff
@@ -151,7 +151,7 @@ def generate_agents(self) -> List:
         return [self.generate_agent(i, idx) for i, idx in agent_override_idx]
 
     def generate_common_vars(self, agent_n: int) -> Dict:
-        """Retrieve vars common for valory apps."""
+        """Retrieve vars common for agent services."""
```
Comment (angrybayblade) [inline autonomy/deploy/base.py:154]:
```suggestion
        """Retrieve vars common for agent."""
```

File: autonomy/deploy/build.py [hunk 5]
```diff
@@ -50,7 +50,7 @@ def generate_deployment(  # pylint: disable=too-many-arguments, too-many-locals
     use_hardhat: bool = False,
     use_acn: bool = False,
 ) -> str:
-    """Generate the deployment build for the valory app."""
+    """Generate the deployment build for the agent service."""
```
Comment (angrybayblade) [inline autonomy/deploy/build.py:53]:
```suggestion
    """Generate the deployment for the service."""
```

File: docs/application_deployment.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
-# Valory Application Deployment
+# Agent service deployment
```
Comment (angrybayblade) [inline docs/application_deployment.md:1]: Same as above

File: README.md (hunk 1 above)
Comment (jmoreira-valory) [reply README.md:43]: @DavidMinarsch please, give your decision, as it seems there is a missunderstanding here, and I will adhere to it consistently. See also the naming convention document.

Comment (DavidMinarsch) [reply README.md:43]: Let's go with the table as discussed @jmoreira-valory

Comment (DavidMinarsch) [reply README.md:43]: (as in with the naming convention document)

Comment (jmoreira-valory) [reply README.md:43]: Ok, then 'agent services'.

**Review by angrybayblade — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**